hardware_interface/include/hardware_interface/handle.hpp

```diff
@@ -15,42 +15,50 @@
 #ifndef HARDWARE_INTERFACE__HANDLE_HPP_
 #define HARDWARE_INTERFACE__HANDLE_HPP_
 
+#include <memory>
```

Comment:
```suggestion
```
I'm not sure about memory header as well as we are not using any smart pointers or allocators. Cross check this one